ssh/tailssh/incubator.go

```diff
@@ -160,11 +164,16 @@ func parseIncubatorArgs(args []string) (a incubatorArgs) {
 	flags.BoolVar(&a.hasTTY, "has-tty", false, "is the output attached to a tty")
 	flags.StringVar(&a.cmdName, "cmd", "", "the cmd to launch (ignored in sftp mode)")
 	flags.BoolVar(&a.isSFTP, "sftp", false, "run sftp server (cmd is ignored)")
+	flags.StringVar(&a.loginCmdPath, "login-cmd", "", "the path to `login` cmd")
 	flags.Parse(args)
 	a.cmdArgs = flags.Args()
 	return a
 }
 
+func (a incubatorArgs) isShell() bool {
+	return len(a.cmdArgs) == 1 && a.cmdArgs[0] == "-l"
```

Comment:
Probably quite false-positivey. `ls -l` matches this.

Pass an incubatorArg for "am I spawning a login shell" ?

Reply:
Done